app/mailers/notice_mailer.rb

```diff
@@ -10,4 +10,11 @@ def remove_trainee user_course
     @course = user_course.course
     mail to: @user.email, subject: t("mailers.remove_trainee.title_subject")
   end
+
+  def statistic_trainee user_course
+    @course = user_course.course
+    @supervisor = @course.suppervisor
```

Comment:
1 course chỉ có 1 suppervisor thôi hả em?

Reply:
Do DB của em vậy á chị. Supervisor của em chia làm  2 loại
1 là thằng có trường group trong table users = 1. Kiểu như là thằng admin . Có toàn quyền

Còn thằng supervisor 2 là lúc tạo course hay subject sẽ được chọn. Quan hệ 1 nhiều thôi chi. Nó chỉ quản lý theo từng course đó thôi ( Thay vì lấy mặc định thằng user tạo course làm khóa ngoại) thì em cho phép chọn 1 thằng user bất kỳ làm supervisor của của course, subject đó.
![db-chithao](https://user-images.githubusercontent.com/47617168/54324624-056ecc80-4631-11e9-9e6e-cf273aa73481.png)